test/hsm/Dockerfile

```diff
@@ -1,11 +1,7 @@
 FROM letsencrypt/boulder-tools:2016-07-05
 MAINTAINER Roland Bracewell Shoemaker <roland@letsencrypt.org>
 
-COPY test-ca.key /root/key.pem
-
-RUN echo "0:/var/lib/softhsm/slot0.db" > /etc/softhsm/softhsm.conf && \
-    softhsm --init-token --slot 0 --label key --pin 1234 --so-pin 0000 && \
-    softhsm --import /root/key.pem --slot 0 --label key --id BEEF --pin 1234
+RUN echo "0:/var/lib/softhsm/slot0.db" > /etc/softhsm/softhsm.conf
```

Comment:
My idea with the previous comment was that we could move this echo, plus the pkcs11-tool invocations, into test/boulder-tools/build.sh, and eliminate test/hsm/Dockerfile entirely.